src/ArborX_LinearBVH.hpp

```diff
@@ -169,15 +177,17 @@ BoundingVolumeHierarchy<MemorySpace, Enable>::BoundingVolumeHierarchy(
       "ArborX::BVH::BVH::calculate_scene_bounding_box");
 
   // determine the bounding box of the scene
+  Box bbox{};
   Details::TreeConstruction::calculateBoundingBoxOfTheScene(space, primitives,
-                                                            _bounds);
+                                                            bbox);
 
   Kokkos::Profiling::popRegion();
 
   if (size() == 1)
   {
     Details::TreeConstruction::initializeSingleLeafNode(
         space, primitives, _internal_and_leaf_nodes);
+    _bounds = getBoundingVolume(getRoot());
```

Comment:
This is a memory access violation.  I am aware of it.  Just didn't bother solving yet.